README.md

```diff
@@ -4,13 +4,14 @@ The Stellar Laboratory is a suite of tools to help one learn about exploring the
 
 ## Developing
 ```
-npm install
+yarn install
 ./node_modules/.bin/gulp develop
 ```
 
 ## Building for production
 ```
-npm install
+yarn
```

Comment:
Shouldn't this be on one line?

Reply:
Oh man, that's why we need reviews 😉. Fixing.